admin-manual/installation-setup/integrations/duracloud-setup.rst

```diff
@@ -1,226 +1,101 @@
-
-  * Under "Other options," create a prefix for your sync folder to create a
-      directory structure at least two levels deep. The prefix must end in a
-      slash (/).
-
-2. Note that the prefix will replace the directory name of the sync folder in
-   DuraCloud.
-
-Example 1:
-
-.. code:: bash
-
-   /syncFolder
-      Digital objects
-
-If the sync directory selected in the sync tool is ``syncFolder``, you could add
-a prefix such as: ``transfers/Project1/``. Archivematica would then recognize
-``transfers`` as a transfer source, and ``Project1`` would be available as a
-transfer. That transfer would contain all of the digital objects in ``syncFolder``.
-
-3. It is also possible to use the prefix option with a sync folder which has
-   subfolders.
-
-Example 2:
-
-.. code:: bash
-
-   /syncFolder
-        /Photographs
-             Digital objects
-        /Videos
-             Digital objects
-
-If the sync directory selected in the sync tool is ``syncFolder``, you could add
-a prefix such as: ``transfers/Project1``. In this example, ``transfers`` will
-still be interpreted by Archivematica as a transfer source, but either
-``Project1``, ``Photographs`` or ``Video`` could be chosen in the dashboard as
-the transfer.
-
-Step 5: Configure transfer sources
-----------------------------------
-
-Assuming that you have configured transfer-source spaces in DuraCloud,
-return to the Archivematica Storage Serice at this point to configure transfer
-sources locations.
-
-Navigate to the transfer-source Storage Service space and create locations for
-each top level directory inside the directory or directories you have synced.
-
-.. tip::
-
-   If you consistently use the same named top-level directory when syncing
-   (e.g. "transfers") you will only need to configure the transfer source
-   once.
-
-.. tip::
-   If you browse for a path and do not see the expected directory listed, this
-   may be caused by a UI bug. You can type the path in instead.
+#. In the Storage Service's Spaces tab, click **Create new space** and fill out
+   the form following the instructions in the :ref:`DuraCloud Storage Service
+   documentation <storageservice:duracloud>`.
+
+#. You must create a space for each storage location that you have created in
+   DuraCloud. For example, if you have AIP storage and DIP storage locations in
+   DuraCloud, you should create two spaces by repeating the instructions in Step
+   1.
+
+#. In the Storage Service, navigated to the **Spaces** tab. You should see all
+   of the DuraCloud spaces that you created in Steps 1 and 2.
+
+#. For each space, you need to create a corresponding :ref:`location
+   <storageservice:locations>`. Click on **Create Location here** to create a
+   storage location and enter the following information:
+
+   * **Purpose**: The purpose of the space (AIP storage, DIP storage, or
+     transfer backlog.)
+   * **Pipeline**: Click on the pipeline name so that it is highlighted orange.
```

Comment:
Accessibility: perhaps we can just say highlighted rather than orange? My own point of reference is that in my dark scheme I see largely in different colors, but other folk might have different barriers. (Also, in regular mode I didn't know this should be orange for me!) 

![image](https://user-images.githubusercontent.com/1880412/73556311-33adf400-441d-11ea-9036-4b7e15c689b6.png)

Reply:
Excellent point!